src/wordy/ast/AssignmentNode.java

```diff
@@ -1,8 +1,11 @@
 package wordy.ast;
```

Comment:
I forgot to delete the throw new UnsupportedOperationException in the abstract main method